doc/source/design.rst

```diff
@@ -5,7 +5,6 @@ The application is based on five entities, as written in the introduction:
 
```

Comment:
I guess we're down to four entities now! But since the KPIs will eventually have their own unique properties (The constraint based stuff) we could still mention them here?
Though I wouldn't know what to write about that at the moment, we could just have a placeholder there until we know exactly what form things will take.